compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -210,19 +212,23 @@ internal class ComposeWindow(
             }
         }
 
+        @Suppress("RedundantOverride")
         override fun convertLocalToWindowPosition(localPosition: Offset): Offset {
-            val (canvasX, canvasY) = getCanvasCoordinates()
-            return Offset(
-                x = (canvasX + localPosition.x / density.density).toFloat(),
-                y = (canvasY + localPosition.y / density.density).toFloat()
-            )
+            // TODO (o.karpovich): Currently, CfW uses AttachedComposeSceneLayer, so
```

Comment:
`AttachedComposeSceneLayer` is name of private inner class, so it's not clear what it is about. Refer to `CanvasLayersComposeScene` instead

Reply:
the note is internal anyway. `AttachedComposeSceneLayer` can be easily found by searching, and I wanted to note its name here so it's what to look at if needed.